taskvine/src/bindings/python3/ndcctools/taskvine/task.py

```diff
@@ -1059,12 +1047,34 @@ def __init__(self, library, fn, *args, **kwargs):
         # function calls at worker only need the name of the function.
         self.set_command(fn)
 
+        self._library = library
```

Comment:
This attribute should not be needed, specially when it could get out of sync with _library_name, and because it could be a string or a function, which make bookkepping hard.